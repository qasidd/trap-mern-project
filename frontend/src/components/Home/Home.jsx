'use strict';
import logo from '../../resources/img/logo.jpg';

const Home =()=>{
return(
    <>
        <h1> Welcome to QA Cinemas! </h1>
        <br></br>
        <img src={logo} alt="this is a logo"/>
        <h2> Redefining the cinema experience </h2>
        <br></br>
        <p> This is the homepage for QA cinemas. This page will provide information about movies, listings and upcoming releases. </p>
        <h3> Page Under Construction </h3>
        <br></br>
        <br></br>
        <br></br>
        <br></br>
        <br></br>
        <br></br>
        <br></br>
        <br></br>
        <br></br>
        <br></br>
</>

);

}

export default Home;