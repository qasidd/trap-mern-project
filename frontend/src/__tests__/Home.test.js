import Home from '../components/Home/Home';
import '@testing-library/jest-dom';
import { expect, it } from '@jest/globals';
import { create } from 'react-test-renderer';

it(`It should match the snapshot every time it's rendered`, () => {
    const instance = create(<Home />).toJSON();
    expect(instance).toMatchSnapshot();
})