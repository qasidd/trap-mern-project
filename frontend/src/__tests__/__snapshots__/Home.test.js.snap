// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should match the snapshot every time it's rendered 1`] = `
Array [
  <h1>
     Welcome to QA Cinemas! 
  </h1>,
  <br />,
  <img
    alt="this is a logo"
    src="logo.jpg"
  />,
  <h2>
     Redefining the cinema experience 
  </h2>,
  <br />,
  <p>
     This is the homepage for QA cinemas. This page will provide information about movies, listings and upcoming releases. 
  </p>,
  <h3>
     Page Under Construction 
  </h3>,
  <br />,
  <br />,
  <br />,
  <br />,
  <br />,
  <br />,
  <br />,
  <br />,
  <br />,
  <br />,
]
`;
